exporter/trace/trace.go

```diff
@@ -44,23 +53,72 @@ func newTraceExporter(o *options) (*traceExporter, error) {
 		client:    client,
 		o:         o,
 	}
+	b := bundler.NewBundler((*contextAndSpans)(nil), func(bundle interface{}) {
+		ctxSpans := bundle.([]*contextAndSpans)
+		ctxToSpansMap := make(map[context.Context][]*tracepb.Span)
+		// upload spans with same context in batch
+		for _, cs := range ctxSpans {
+			ctxToSpansMap[cs.ctx] = append(ctxToSpansMap[cs.ctx], cs.spans...)
+		}
+		for ctx, spans := range ctxToSpansMap {
+			e.uploadFn(ctx, spans)
+		}
+	})
+	if o.BundleDelayThreshold > 0 {
+		b.DelayThreshold = o.BundleDelayThreshold
+	} else {
+		b.DelayThreshold = 2 * time.Second
+	}
+	if o.BundleCountThreshold > 0 {
+		b.BundleCountThreshold = o.BundleCountThreshold
+	} else {
+		b.BundleCountThreshold = 50
+	}
+	if o.NumberOfWorkers > 0 {
+		b.HandlerLimit = o.NumberOfWorkers
```

Comment:
Based on the description https://github.com/googleapis/google-api-go-client/blob/cfc62336a21b9af45e2830a9898569d8aac1c5cc/support/bundler/bundler.go#L80-L82, the config parameter should be called `MaxNumberOfWorkers` rather than just `NumberOfWorkers`, or Just `BundleHandlerLimit` to directly refer to the bundler parameter.